Mage/src/main/java/mage/game/combat/Combat.java

```diff
@@ -1659,6 +1664,25 @@ public UUID getDefenderId(UUID attackerId) {
      * @return
      */
     public UUID getDefendingPlayerId(UUID attackingCreatureId, Game game) {
+        return getDefendingPlayerId(attackingCreatureId, game, false);
+    }
+
+    /**
+     * Returns the playerId of the player that is attacked by given attacking
+     * creature or formerly-attacking creature.
+     *
+     * @param attackingCreatureId
+     * @param game
+     * @return
+     */
+    public UUID getDefendingPlayerId(UUID attackingCreatureId, Game game, boolean allowFormer) {
+        if (allowFormer) {
```

Comment:
Must have 802.2a rules text here or it's part (about founding of defending player). Is it can be a default logic (allowFormer = true)? Looks like rules used it for anything.

Reply:
Done, made it default logic - there's two that actually need to be false (one uses this method for buffing attacking creatures and the other is "as long as {this} is attacking, defending player can't cast spells")